style.css

```diff
@@ -0,0 +1,136 @@
+*{
+    margin:0;
+    padding:0;
+}
+header img{
+    height: 70px;
```

Comment:
Always give `max-width` or `max-height` to images. In this case, if the new image is 50px in height - it'll unnecessarily stretch to 70px and look blurry.